documentation/bolt_installing_modules.md

```diff
@@ -329,9 +332,98 @@ module-install:
 - [bolt-project.yaml options](bolt_project_reference.md#module-install)
 - [bolt-defaults.yaml options](bolt_defaults_reference.md#module-install)
 
+## Skip dependency resolution for a module
+
+Sometimes you might want to skip dependency resolution for a specific module,
+usually because Bolt is unable to resolve its dependencies. This allows you to
+take advantage of Bolt's module dependency resolution while still using modules
+that Bolt cannot resolve dependencies for. You might want to skip dependency
+resolution for a module if:
+
+- The module has outdated or incorrect metadata.
+- The module is a git module hosted in a repository other than a public GitHub
+  repository.
+
+You can configure Bolt to skip dependency resolution for a module by setting the
+`resolve` key for the module specification to `false`. This key is available for
+both Forge and git modules.
+
+When you install modules with Bolt, it only resolves module dependencies for
+module specifications that do not set `resolve: false`. After resolving
+dependencies, Bolt generates a Puppetfile with the resolved modules and
+dependencies, as well as the modules it did not resolve dependencies for.
+
+For example, if your project includes the `puppetlabs/ntp` Forge module and a
+git module named `private_module` hosted in a private GitHub repository, you can
+configure Bolt to skip dependency resolution for `private_module`:
+
+```yaml
+# bolt-project.yaml
+name: myproject
+modules:
+  - puppetlabs/ntp
+  - name: private_module
+    git: git@github.com:puppetlabs/private_module
+    ref: 1.0.0
+    resolve: false
+```
+
+> **Note:** When setting `resolve: false` for a git module, you must include
```

Comment:
I think it's also worth calling out (here or elsewhere) that the value of `name` is how the module will be named on the modulepath as well.

Reply:
Added some more info to the table further up the page that lists the available keys.